<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightingDialog</class>
 <widget class="QDialog" name="LightingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lighting</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLights">
      <attribute name="title">
       <string>Lights</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0,1">
       <property name="horizontalSpacing">
        <number>18</number>
       </property>
       <item row="1" column="2">
        <widget class="QLabel" name="labelAuxiliaryLightPosition">
         <property name="text">
          <string>(-0.0, -0.0, -0.0)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSlider" name="sliderY">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxHeadlight">
         <property name="text">
          <string>Headlight</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxAuxiliaryLight">
         <property name="text">
          <string>Auxiliary Light</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Auxiliary Light Dir.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QSlider" name="sliderAuxiliaryLightIntensity">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QSlider" name="sliderHeadlightIntensity">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSlider" name="sliderX">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSlider" name="sliderZ">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelHeadlightIntensity">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelAuxiliaryLightIntensity">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSurface">
      <attribute name="title">
       <string>Surface</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2">
        <widget class="QLabel" name="labelSpecularPower">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSlider" name="sliderAmbient">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSlider" name="sliderDiffuse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelAmbient">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelDiffuse">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelSpecular">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Specular Power:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSlider" name="sliderSpecularPower">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSlider" name="sliderSpecular">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Specular:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Ambient:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Diffuse:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBackground">
      <attribute name="title">
       <string>Background</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="6">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>38</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Green:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Blue:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Red:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelRed">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelGreen">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelBlue">
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="4">
        <widget class="QSlider" name="sliderRed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="4">
        <widget class="QSlider" name="sliderGreen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="4">
        <widget class="QSlider" name="sliderBlue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QRadioButton" name="radioButtonWhite">
         <property name="text">
          <string>White</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="3">
        <widget class="QRadioButton" name="radioButtonCustom">
         <property name="text">
          <string>Custom</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDefault">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDone">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBoxHeadlight</tabstop>
  <tabstop>sliderHeadlightIntensity</tabstop>
  <tabstop>checkBoxAuxiliaryLight</tabstop>
  <tabstop>sliderAuxiliaryLightIntensity</tabstop>
  <tabstop>sliderX</tabstop>
  <tabstop>sliderY</tabstop>
  <tabstop>sliderZ</tabstop>
  <tabstop>sliderDiffuse</tabstop>
  <tabstop>sliderAmbient</tabstop>
  <tabstop>sliderSpecular</tabstop>
  <tabstop>sliderSpecularPower</tabstop>
  <tabstop>radioButtonWhite</tabstop>
  <tabstop>radioButtonCustom</tabstop>
  <tabstop>sliderRed</tabstop>
  <tabstop>sliderGreen</tabstop>
  <tabstop>sliderBlue</tabstop>
  <tabstop>pushButtonDefault</tabstop>
  <tabstop>pushButtonDone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
